server-manager/server-manager.csproj

```diff
@@ -2,11 +2,21 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Seems unnecessary, why this upgrade? I think it's better to try to avoid the short term releases (even though this was at 5 hehe). I would prefer 6 or 8